taipy/core/job/job.py

```diff
@@ -58,7 +60,7 @@ class Job(_Entity, _Labeled):
     _MANAGER_NAME = "job"
     _ID_PREFIX = "JOB"
 
-    def __init__(self, id: JobId, task: Task, submit_id: str, submit_entity_id: str, force=False, version=None):
+    def __init__(self, id: JobId, task: "Task", submit_id: str, submit_entity_id: str, force=False, version=None):
```

Comment:
I think we could eventually avoid that using `from __future__ import annotations`.

Reply:
I didn't know about that! Certainly sounds really useful! Maybe we can have a ticket to approach this later @jrobinAV